docs/articles/documentation/test-api/actions/select-text.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: docs
+title: Select Text
+permalink: /documentation/test-api/actions/select-text.html
+---
+# Select Text
+
+There are different kinds of the Select Text action for selecting regular text, `<textarea>` and `contentEditable` content.
+
+* [Select Regular Text](#select-regular-text)
+* [Select \<textarea\> Content](#select-textarea-content)
+* [Perform Selection within Editable Content](#perform-selection-within-editable-content)
+
+## Select Regular Text
+
+```text
+t.selectText( selector, startPos, endPos )
+```
+
+Parameter  | Type                                              | Description
+---------- | ------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------
+`selector` | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the webpage element whose text will be selected. See [Selecting Target Elements](index.md#selecting-target-elements).
+`startPos` | Number                                            | The start position of the selection.
+`endPos`   | Number                                            | The end position of the selection.
+
+The following example demonstrates text selection within the input element.
+
+```js
+import { expect } from 'chai';
+
+fixture `My fixture`
+    .page('http://www.example.com/');
+
+test('Select text within input', async t => {
+    await t
+        .typeText('#developer-name', 'Test Cafe', { caretPos: 0 })
+        .selectText('#developer-name', 7, 1);
+
+    const input = await t.select('#developer-name');
+
+    expect(input.attributes['selectionStart']).to.equal(1);
+    expect(input.attributes['selectionEnd']).to.equal(7);
```

Comment:
The example doesn't work if you use it for a page with an input because an Input element doesn't have `selectionStart` and `selectionEnd` **attributes**, it has **fields** with these names
I suggest to use something like the following:
```js
const getElementSelectionStart = ClientFunction(id => document.getElementById(id).selectionStart);
const getElementSelectionEnd = ClientFunction(id => document.getElementById(id).selectionEnd);

expect(await getElementSelectionStart('#developer-name')).to.equal(1);
expect(await getElementSelectionEnd('#developer-name')).to.equal(7);
```